dist/aframe-teleport-controls.js

```diff
@@ -132,19 +132,7 @@
 
 	    this.onButtonDown = this.onButtonDown.bind(this);
 	    this.onButtonUp = this.onButtonUp.bind(this);
-	    if (this.data.startEvents.length && this.data.endEvents.length) {
-
-	      for (i = 0; i < this.data.startEvents.length; i++) {
-	        el.addEventListener(this.data.startEvents[i], this.onButtonDown);
-	      }
-	      for (i = 0; i < this.data.endEvents.length; i++) {
-	        el.addEventListener(this.data.endEvents[i], this.onButtonUp);
-	      }
-	    } else {
-	      el.addEventListener(data.button + 'down', this.onButtonDown);
-	      el.addEventListener(data.button + 'up', this.onButtonUp);
-	    }
-
```

Comment:
The fact that I did away with these lines mean the "startEvents" and "endEvents" properties don't do anything anymore. They might be un-necessary now with the new WebXR api, but I wasn't sure. Thought it was worth noting, and is part of the reason I didn't originally create this pull request.

Reply:
I don't think is a good idea to remove these events and replace them just with `axismove` as you are forcing the users to use the axis for teleporting without any chance to change that configuration.

Reply:
I added this back, but switched the default behavior to axismove as I believe that is the most common desired behavior (due to quest).